cni/ipam/ipam_test.go

```diff
@@ -4,17 +4,20 @@
 package ipam
 
 import (
+	"encoding/json"
 	"fmt"
+	"github.com/Azure/azure-container-networking/common"
+	"github.com/Azure/azure-container-networking/platform"
+	cniSkel "github.com/containernetworking/cni/pkg/skel"
+	cniTypesCurr "github.com/containernetworking/cni/pkg/types/current"
+	. "github.com/onsi/ginkgo"
```

Comment:
we don't use submodules in this repo so please vendor via go-dep

Reply:
I'm now using go-dep instead, adding github.com/onsi/ginkgo, github.com/onsi/gomega and gopkg.in/fsnotify.v1